backend/application/syson-sysml-export/src/main/java/org/eclipse/syson/sysml/export/utils/NameDeresolver.java

```diff
@@ -140,7 +141,8 @@ private String deresolve(Element element, Namespace sourceNamespace, Namespace d
             if (cacheValue != null) {
                 return cacheValue;
             }
-            EList<Membership> visibleMemberships = this.getVisibleMemberships(deresolvingNamespace, deresolvingNamespace == sourceNamespace);
+            boolean includePrivate = deresolvingNamespace == sourceNamespace || EcoreUtil.isAncestor(deresolvingNamespace, sourceNamespace);
```

Comment:
Do not add boolean expressions with the following format: `x == y || z`.
It's not readable.
Please break operands down, at least in two.

Reply:
done.